package.json

```diff
@@ -49,10 +49,9 @@
   "dependencies": {
     "@asyncapi/avro-schema-parser": "^0.2.0",
     "@asyncapi/openapi-schema-parser": "^2.0.0",
-    "@asyncapi/parser": "v1.0.1",
+    "@asyncapi/parser": "1.1.0",
```

Comment:
Shouldn't we do this in a separate release before? To make sure it works.

Reply:
yeah, I thought about this, but I don't really think it is needed, 1.1.0 just added one additional validation of the AsyncAPI files, the query params in channels. This should not have any effect on the generator itself, even if there are bugs, they are not breaking the compatibility of the generator